src/hotspot/share/runtime/arguments.cpp

```diff
@@ -336,6 +336,17 @@ bool Arguments::is_internal_module_property(const char* property) {
   return false;
 }
 
+// Return true if the key matches the --module-path property name ("jdk.module.path").
+bool Arguments::is_module_path_property(const char* key) {
+  if (strncmp(key, MODULE_PROPERTY_PREFIX, MODULE_PROPERTY_PREFIX_LEN) == 0) {
+    const char* property_suffix = key + MODULE_PROPERTY_PREFIX_LEN;
+    if (matches_property_suffix(property_suffix, PATH, PATH_LEN)) {
+      return true;
+    }
+  }
+  return false;
```

Comment:
Can this be simplified to `return (strcmp(key, MODULE_PROPERTY_PREFIX PATH) == 0)`?

Reply:
I'm not sure. Is your suggest equivalent to:
  `return (strcmp(key, "jdk.module.path"));`

Reply:
Yes, the C++ compiler will automatically concatenate `MODULE_PROPERTY_PREFIX PATH` into a single string.

Reply:
Done.